buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -103,6 +104,8 @@
         DEFAULT_CACHE_TRIM_INTERVAL = SystemPropertyUtil.getInt(
                 "io.netty.allocator.cacheTrimInterval", 8192);
 
+        DEFAULT_MAX_POOLED_BYTES = SystemPropertyUtil.getLong("io.netty.allocator.maxPooledBytes", Long.MAX_VALUE);
```

Comment:
If this setting is per arena, then I think the system property should be named accordingly (such as ```io.netty.allocator.arena.maxPooledBytes```), otherwise this property gives the impression that the limit is for the entire allocator.